adaptive/learner/integrator_coeffs.py

```diff
@@ -141,39 +142,48 @@ def calc_V(x, n):
     return np.array(V).T
 
 
-eps = np.spacing(1)
+@lru_cache(maxsize=None)
+def _coefficients():
```

Comment:
One last request: let us provide a minimal docstring explaining the purpose of this function. Something like "Compute the coefficients on demand, in order to avoid doing linear algebra on import."

Reply:
Aside of this everything looks good!

Reply:
Great! I have just added that description.